plugins/generator-1.15.2/forge-1.15.2/procedures/world_data_is_block_side_solid.java.ftl

```diff
@@ -0,0 +1 @@
+(Block.hasSolidSide(world.getBlockState(new BlockPos((int)${input$x}, (int)${input$y}, (int)${input$z})), world, new BlockPos((int)${input$x}, (int)${input$y}, (int)${input$z}), ${input$direction}))
```

Comment:
Block related procedures are in blocks category right now so I think this block should be there too. Maybe a new category for block data would be appropriate?

Reply:
The block was already in that category, though I renamed it to make it clearer